### PR TITLE
Task/python3

### PR DESCRIPTION
**Pull request by sapnamysore**
Python3, Django 2 updates

### CODE REVIEW

File: canvas_manage_course/settings/base.py [hunk 1]
```diff
@@ -275,6 +264,11 @@
             'level': _DEFAULT_LOG_LEVEL,
             'propagate': False,
         },
+        'django-ssm-parameter-store': {
+            'handlers': ['default', 'console'],
+            'level': _DEFAULT_LOG_LEVEL,
+            'propagate': False,
+        },
```
Comment (Chris-Thornton-Harvard) [inline canvas_manage_course/settings/base.py]: I forget but did we decide on keeping this logging and if so, do we want the console handler?

File: canvas_manage_course/views.py [hunk 2]
```diff
@@ -80,17 +78,15 @@ def lti_launch(request):
 @login_required
 @lti_permission_required('canvas_manage_course')
 def dashboard_course(request):
-    course_instance_id = request.LTI.get('lis_course_offering_sourcedid')
```
Comment (Chris-Thornton-Harvard) [inline canvas_manage_course/views.py]: Not sure this should be deleted

File: canvas_manage_course/views.py [hunk 3]
```diff
@@ -80,17 +78,15 @@ def lti_launch(request):
...
     tool_access_permission_names = [
         'class_roster',
-        'im_import_files',  # isites_migration
         'manage_people',
         'manage_sections',
         'custom_fas_card_1']
 
     # Verify current user permissions to see the apps on the dashboard
-    allowed = {tool: lti_permission_required_check(request, tool)
+    allowed = {tool: True
```
Comment (Chris-Thornton-Harvard) [inline canvas_manage_course/views.py]: Should be changed back to its original state

File: manage_sections/views.py [hunk 4]
```diff
@@ -412,8 +421,8 @@ def add_to_section(request):
     })
 
 
-@login_required
-@lti_permission_required('manage_sections')
+# @login_required
+# @lti_permission_required('manage_sections')
```
Comment (Chris-Thornton-Harvard) [inline manage_sections/views.py]: There are a handful of @login_required and @lti_permission_required decorators commented out

File: manage_sections/views.py [hunk 5]
```diff
@@ -179,9 +187,10 @@ def create_section_form(request):
                 if is_sis_section(sis_section_id):
                     section['registrar_section_flag'] = True
                 section_list.append(section)
+        print('Done going through sections getting count')
 
         # case insensitive sort the sections in alpha order
-        section_list = sorted(section_list, key=lambda x: x[u'name'].lower())
+        # section_list = sorted(section_list, key=lambda x: x[u'name'].lower())
```
Comment (Chris-Thornton-Harvard) [inline manage_sections/views.py]: Uncomment and remove the unicode in name

File: manage_sections/views.py [hunk 6]
```diff
@@ -164,7 +163,16 @@ def create_section_form(request):
                 'No sections found for Canvas course %s' % canvas_course_id
             )
             return render(request, 'manage_sections/error.html', status=500)
+        print('Going through canvas sections')
         for section in canvas_sections:
+            print('preparing enrollment count')
+            # if section.get('enrollments'):
+            #     section['enrollment_count'] = len(_filter_student_view_enrollments(section['enrollments']))
```
Comment (Chris-Thornton-Harvard) [inline manage_sections/views.py]: Remove commented out sections

File: manage_sections/views.py [hunk 7]
```diff
@@ -164,7 +163,16 @@ def create_section_form(request):
...
+            if section.get('enrollments'):
+                section['enrollment_count'] = len(_filter_student_view_enrollments(section['enrollments']))
+            else:
+                section['enrollment_count'] = 'n/a'
+            print('Done with enrollment count')
```
Comment (Chris-Thornton-Harvard) [inline manage_sections/views.py]: There are a few print statements in this file

File: selenium_tests/course_admin/course_admin_base_test_case.py [hunk 8]
```diff
@@ -39,4 +39,4 @@ def setUpClass(cls):
         if login_page.is_loaded():
             login_page.login_xid(cls.USERNAME, cls.PASSWORD)
         else:
-            print('(User {} already logged in to PIN)'.format(cls.USERNAME))
+            print(('(User {} already logged in to PIN)'.format(cls.USERNAME)))
```
Comment (Chris-Thornton-Harvard) [inline selenium_tests/course_admin/course_admin_base_test_case.py]: Unfortunately, 2to3 will keep wrapping print statements in parens. You can remove the outermost parens

File: canvas_manage_course/views.py (hunk 2 above)
Comment (sapnamysore) [reply canvas_manage_course/views.py]: The variable is unused in this method. The subsequent method need it, the  dashboard doesnt use it.

File: canvas_manage_course/settings/base.py (hunk 1 above)
Comment (sapnamysore) [reply canvas_manage_course/settings/base.py]: We didn't keep it in caat. removed here as well

File: canvas_manage_course/views.py [hunk 9]
```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
 
 import logging
-import urllib
-import urlparse
+import urllib.request, urllib.parse, urllib.error
+import urllib.parse
```
Comment (cmurtaugh) [inline canvas_manage_course/views.py]: Can remove the `import urllib.parse` on line 5 as it's being imported in line 4.

File: manage_sections/views.py [hunk 10]
```diff
@@ -122,6 +119,7 @@ def _add_badge_label_name_to_enrollments(enrollments):
 @lti_permission_required('manage_sections')
 @require_http_methods(['GET'])
 def create_section_form(request):
+    print('IN CREATE SECTION!!')
```
Comment (cmurtaugh) [inline manage_sections/views.py]: Maybe change this to a logger or remove it?

File: manage_sections/views.py [hunk 11]
```diff
@@ -164,7 +163,12 @@ def create_section_form(request):
                 'No sections found for Canvas course %s' % canvas_course_id
             )
             return render(request, 'manage_sections/error.html', status=500)
+        print('Going through canvas sections')
         for section in canvas_sections:
+            if section.get('enrollments'):
```
Comment (cmurtaugh) [inline manage_sections/views.py]: This seems to duplicate the `if` below.

File: manage_sections/views.py [hunk 12]
```diff
@@ -276,8 +280,8 @@ def edit_section(request, section_id):
                   {'section': course_section})
 
 
-@login_required
-@lti_permission_required('manage_sections')
+# @login_required
+# @lti_permission_required('manage_sections')
```
Comment (cmurtaugh) [inline manage_sections/views.py]: Should these be uncommented?

File: manage_sections/views.py (hunk 10 above)
Comment (cmurtaugh) [reply manage_sections/views.py]: There are a few more `print()` statements below that should also be removed or converted to loggers.

File: manage_sections/views.py (hunk 12 above)
Comment (sapnamysore) [reply manage_sections/views.py]: Yes, I cleaned up a bunch of places. missed this!

File: manage_sections/views.py (hunk 11 above)
Comment (sapnamysore) [reply manage_sections/views.py]: That's right! thanks. The second if was put when we changed the call for the timeout for large sections.

File: manage_sections/views.py (hunk 10 above)
Comment (sapnamysore) [reply manage_sections/views.py]: cleaned up the prints

**Comment by sapnamysore** [comment]
@cmurtaugh : handled most of your comments. Only manage_sections urls needed re_path vs path. Most other url transformations seem straightforward as these tools don't use angular.

**Comment by cmurtaugh** [comment]
Good find! Looks good to me :+1:

File: canvas_manage_course/settings/base.py [hunk 13]
```diff
@@ -194,6 +186,8 @@
 
 _DEFAULT_LOG_LEVEL = SECURE_SETTINGS.get('log_level', logging.DEBUG)
 _LOG_ROOT = SECURE_SETTINGS.get('log_root', '')  # Default to current directory
+_LOG_ROOT= ''
```
Comment (Chris-Thornton-Harvard) [inline canvas_manage_course/settings/base.py]: Should this be included?

Comment (sapnamysore) [reply canvas_manage_course/settings/base.py]: Nope, I think it was a local override. good catch! will clean it up on develop directly

**Pull request merged**